tests/image-magick/read-collection.spec.ts

```diff
@@ -53,8 +53,8 @@ describe('ImageMagick#readCollection', () => {
                 await bogusAsyncMethod();
             });
         })
-        .rejects
-        .toThrowError('ReadPNGImage');
+            .rejects
```

Comment:
This looks indented too much?

Reply:
commit ac5a0b4d78665beb494217032bd8a71722dfe5c0.
Thanks to let me know